app/setup_pf.py

```diff
@@ -116,6 +116,8 @@ def setup_configs():
         seek_query_form_custom_htmls={'en': '', 'fr': ''},
         published_date=get_timestamp(datetime(2010, 1, 12)),
         updated_date=get_timestamp(datetime(2010, 1, 12)),
+        admin_user_email=get_current_user_email(),
```

Comment:
I think admin_user_email is not appropriate for the name of this variable. Thus I will change the name to unreviewed_notes_email and add default value to ''.

Thanks!